### DISCUSSION TITLE
How stable is V3?

### DISCUSSION THREAD

**Discussion by TheCutter** [bertmelis/VitoWiFi#108] [General]
Is it possible for a new project to start with V3 or are some core parts still missing?

**Comment by bertmelis** [comment]
I have it non-stop running on a raspberry pi for several weeks now so I would say it is ready to release.

Therefore I also don't recommend using the other versions. My focus is only on v3.

What is missing? Type conversions. But then again, it is easy to access the raw data so you can encode/decode in your own code. When the conversion is added to the library, you can still switch or do nothing and keep your own code.

**Comment by TheCutter** [comment]
Okay thank you. I will give it a try.

**Comment by TheCutter** [comment]
It runs really well. Good work!
